### PR TITLE
Records and ID show gender identity instead of sex

### PR DESCRIPTION
**Pull request by MortalRed**
Records will now show gender identity instead of assigned sex at birth.

### CODE REVIEW

**Comment by MortalRed** [comment]
!review

**Comment by NonQueueingMatt** [comment]
Wouldn't it make sense to just disclose gender instead? Not everyone has records, and these basic fields are meant to provide the very basic details.

**Comment by MortalRed** [comment]
That was the original intent, but because the relevant dm files are in completely different locations, I wasn't able to rely on calling get_pronoun() like I was able to last time.

**Comment by NonQueueingMatt** [comment]
> That was the original intent, but because the relevant dm files are in completely different locations, I wasn't able to rely on calling get_pronoun() like I was able to last time.

I'm not sure I follow the reasoning here, there's no reason you can't use get_pronoun. If you're confused on how to use it I can help you there.

File: code/game/objects/items/weapons/cards_ids.dm [hunk 1]
```diff
@@ -178,7 +178,6 @@ var/const/NO_EMAG_ACT = -50
 /obj/item/card/id/proc/dat()
 	var/dat = ("<table><tr><td>")
 	dat += text("Name: []</A><BR>", registered_name)
-	dat += text("Sex: []</A><BR>\n", sex)
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/cards_ids.dm:181]: Needs gender.

File: code/modules/paperwork/filingcabinet.dm [hunk 2]
```diff
@@ -116,7 +116,6 @@
 				P.info = "<CENTER><B>Security Record</B></CENTER><BR>"
 				P.info += {"
 Name: [R.name] ID: [R.id]<BR>
-Sex: [R.sex]<BR>
```
Comment (NonQueueingMatt) [inline code/modules/paperwork/filingcabinet.dm:119]: Needs gender.

File: code/modules/paperwork/filingcabinet.dm [hunk 3]
```diff
@@ -158,7 +157,6 @@ Important Notes:<BR>
 				var/info = "<CENTER><B>Medical Record</B></CENTER><BR>"
 				info += {"
 Name: [R.name] ID: [R.id]<BR>
-Sex: [R.sex]<BR>
```
Comment (NonQueueingMatt) [inline code/modules/paperwork/filingcabinet.dm:161]: Needs gender.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by MortalRed** [comment]
Oh wow it's been how long since I've last looked at this?
Gonna close this one, get an update, clean some thing up, figure out how on Earth I'm supposed to get this to work with IDs, and so on.
It'll be back Soon:tm:

**Pull request closed without merging**